sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -481,6 +481,15 @@ pi_result piextPlatformCreateWithNativeHandle(pi_native_handle, pi_platform *) {
   DIE_NO_IMPLEMENTATION;
 }
 
+__SYCL_EXPORT pi_result piextGetMemoryConnection(pi_device device1, pi_context context1, pi_device device2, pi_context context2, memory_connection* res){
+  (void) device1;
+  (void) context1;
+  (void) device2;
+  (void) context2;
+  *res = MEMORY_CONNECTION_NONE;
```

Comment:
What if device1/context1 is the same as device2/context2?

Reply:
This case is taken care of in the SYCL runtime. This function is only called if either contexts or devices are not the same. See `getMemoryConnection` in `graph_builder.cpp`.

Reply:
NIT: maybe assert then?